service/history/workflowExecutionContext.go

```diff
@@ -944,7 +860,7 @@ func (c *workflowExecutionContextImpl) update(
 	c.msBuilder.GetExecutionInfo().LastUpdatedTimestamp = c.timeSource.Now()
 
 	// for any change in the workflow, send a event
-	c.shard.NotifyNewHistoryEvent(newHistoryEventNotification(
+	_ = c.shard.NotifyNewHistoryEvent(newHistoryEventNotification(
```

Comment:
will this pass lint check?

Reply:
NotifyNewHistoryEvent function can return error, however in this case, the error should not be returned since workflow change notification is not really important.

BTW `_ = funcation call` is to make the linter happy